databricks/koalas/frame.py

```diff
@@ -207,6 +207,76 @@ def iteritems(self):
         cols = list(self.columns)
         return list((col_name, self[col_name]) for col_name in cols)
 
+    def to_clipboard(self, excel=True, sep=None, **kwargs):
+        """
```

Comment:
pandas does use the r to do raw string here. Otherwise I think you need to escape the backslashes.

Reply:
Nice catch!